docs/content/users/quickstart.md

```diff
@@ -70,6 +70,33 @@ Please note that you will need to change the PHP version to 7.4 to be able to wo
     ddev launch
     ```
 
+## CiviCRM (Standalone)
+
+[CiviCRM Standalone](https://civicrm.org/blog/ufundo/next-steps-civicrm-standalone) allows running [CiviCRM](https://civicrm.org/) without a CMS.
```

Comment:
I think this link should be to https://docs.civicrm.org/installation/en/latest/standalone/

Oddly, those docs don't seem to be ready at all, with no download link and no composer instructions. A composer install would be more in line with most of the other quickstarts.

```suggestion
[CiviCRM Standalone](https://docs.civicrm.org/installation/en/latest/standalone/) allows running [CiviCRM](https://civicrm.org/) without a CMS.
```

Reply:
I linked to the blog post, to offer people some background. There's a link to the docs at the bottom.

Reply:
I didn't find it in the blog and still don't find it after re-reviewing. Please update with links to docs and you can keep the blog link if you think it's useful. It's a pretty rambling piece and makes the project look very, very young :)

Reply:
It's right there at the bottom of the Quick Start :)

> Visit [Install CiviCRM (Standalone)](https://docs.civicrm.org/installation/en/latest/standalone) for more installation details.

- https://github.com/ddev/ddev/pull/6846/files#diff-cdacb5ab99042792ddb67ee7d62ac1d36c4699992949005880517b4955e10f7eR98

Reply:
OK, I thought you were saying it was at the bottom of that blog. Please move the link to the top section, and reference the blog if you'd like. People don't read anything that far down on the page.

Reply:
All right, it's at the top now.